### DISCUSSION TITLE
Global stagger phase limit

### DISCUSSION THREAD

**Discussion by nbaird** [ericaltendorf/plotman#484] [Ideas]
Stagger phases are limited per tmp directory in the config file with tmpdir_stagger_phase_limit but there doesn't seem to exist a global_stagger_phase_limit for example when using 1 tmp drive per plot.
A global limiter would make it possible to regulate threads used on the cpu when using multiple tmp drives.

**Comment by altendky** [comment]
How about we implement a simple global thread limit instead?

**Reply by nbaird** [reply]
Yes good idea that would work too.
